dataspace-connector/full/docker-compose.yml

```diff
@@ -56,9 +56,31 @@ services:
       - configmanager
     restart: always
 
+  keycloak:
+    depends_on:
+      - postgres
+    container_name: local_keycloak
+    environment:
+      DB_VENDOR: postgres
+      DB_ADDR: postgres
+      DB_DATABASE: connectordb
+      DB_USER: connector
+      DB_PASSWORD: 12345
+      KEYCLOAK_USER: admin
+      KEYCLOAK_PASSWORD: admin
+    image: jboss/keycloak:13.0.0

```

Comment:
- Please put the _image:_ on top
- Why 13.0.0 and not latest (14?) https://hub.docker.com/r/jboss/keycloak/tags?page=1&ordering=last_updated

Reply:
Done